src/js/helpers/prepare.js

```diff
@@ -1,11 +1,22 @@
 /* Clearing variables */
 
 import calculateOffset from './calculateOffset';
+import getInlineOption from './getInlineOption';
 
 const prepare = function ($elements, options) {
   $elements.forEach((el, i) => {
+    const positionIn = calculateOffset(el.node, options.offset);
+    const positionOut = positionIn + window.innerHeight;
```

Comment:
So, if I understood well, your proposed solution for `positionOut` would be something like: `(elementOffsetTop + el.offsetHeight) - additionalOffset`?

Shouldn’t this calculation be inside the `calculateOffset` function? Maybe instead of returning a single integer, the function could return an object with both `positionIn` and `positionOut` values.

What do you think?